blog/models.py

```diff
@@ -1,3 +1,19 @@
 from django.db import models
 
 # Create your models here.
+
+class Blog(models.Model):
```

Comment:
nice, seems you did not make and/or commit the migration file, I did it for you quickly pushing the change to this branch:

<img width="1496" alt="image" src="https://user-images.githubusercontent.com/387927/133091766-5bec5ca5-2ec0-4b34-b16c-b2708afc8d60.png">

Reply:
Is that the 0001_initial.py file that sits in the migration folder? If so, I have it, I wonder why it won't commit though.

Reply:
yep that one, hm weird? so it did not show up with `git status`?